vuepress/docs/v6.3.2/docs/releases/README.md

```diff
@@ -1,7 +1,93 @@
+    -  Note that the Component Generator doesn't support this descriptor version yet, but plugins can be easily adapted manually
+  - Introduced the concept of "System Level Bundles", which allows the installation of root level resources.
+  - Added  support the CMS categories
+  - Initial implementation of the bundle Installation plan.
+      - The Installation plan allows to inspect the content of a bundle and to resolve conflicts with the components already present in the destination EntandoApp
+  - Fixed support for bundles based on mysql
+  - Fixed bug that in some condition prevented the bundle plugin pod to properly restart when scaled down and up again
+  - Fixed intermittent issue on the operator that prevented installing new bundles until the operator was restarted
+
+### Entando Component Generator
+
+  - Fixed support for the plugin descriptor ingressPath
+  - Fixed native builds in the Quarkus blueprint
+
+### App Builder
+  - Fixed a couple of permission issues related to page editing.
+  - We started a process of optimization to improve the performance of AppBuilder on scenarios with a high number of pages and contents.
+  - The process will likely be completed for the next minor or path release (6.4.0 or 6.3.3)
+  - Added support for user Avatars
+  - Fixed the CONTENT SEARCH QUERY WIDGET which in some condition returned no results
+  - Several fixes in assets, content, page and user management
+
+### Entando CLI "ent"
+
+ - Ent prj to support profiles in backend build
+ - Ent prj to force semver on bundle versions (X.Y.Z or vX.Y.Z), in accordance with the rest of the installation process
+ - Ent prj now allows direct deployment of a bundle via `ent prj deploy`
+ - Implemented support for connection profiles and related connection to clusters via Kubernetes configuration files and/or context
+ - Implemented a set of ECR operations for
+    - Reading the list of bundles
+    - Installing a bundle in standard mode
+    - Installing a bundle in override mode (override the components already present)
+    - Uninstalling a bundle
+    - Purging a bundle (completely removes the bundle pods and related data)
+
+### Google Kubernetes Engine (GKE)
+  - Tested and validated GKE deployment under new database scenariosImproved the support for GKE that has been tested under several DB and scenarios.
+  - Added documentation for using Google Cloud File for clustered storage
+  - Added documentation and testing for TLS scenarios in Google
+
+## Certifications
+
+### RedHat Certification
+  - Entando 6.3.2 operator is now certified REDHAT. The operator is available in the operator hub of OpenShift 4.X.
+
+### Tanzu Kubernetes Grid (TKG) Certification
+
+  - Entando 6.3.2 operator is now certified Tanzu Kubernetes Grid and can be accessed here: https://marketplace.cloud.vmware.com/services/details/entando11?slug=true
```

Comment:
I think we need the anchor markup on the link.

Reply:
Fixed